### ISSUE TITLE
Cannot run mark2 config

### ISSUE THREAD

**Issue by Yuri20** [gsand/mark2#10]
When I type: python2 mark2 config I get:

    Traceback (most recent call last):
      File "./mark2", line 6, in <module>
        sys.exit(main())
      File "/home/jerzy/mc/mark2/mk2/launcher.py", line 611, in main
        command.start()
      File "/home/jerzy/mc/mark2/mk2/launcher.py", line 126, in start
        self.run()
      File "/home/jerzy/mc/mark2/mk2/launcher.py", line 485, in run
        subprocess.call([editor, path_tmp])
      File "/usr/lib/python2.7/subprocess.py", line 522, in call
        return Popen(*popenargs, **kwargs).wait()
      File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
        errread, errwrite)
      File "/usr/lib/python2.7/subprocess.py", line 1335, in _execute_child
        raise child_exception
    OSError: [Errno 2] No such file or directory

**Comment by gsand** [comment]
Where did you install it to?

Normally, one would install it to `/usr/mark2` or `/usr/local/share/mark2` and symlink to `/usr/local/bin/mark2` so it's able to be ran using `mark2 start /path/to/server` or just `mark2 start`

**Comment by Yuri20** [comment]
I didn't install it, I just went to the folder when it was clonned and typed python2 mark2 config. I also tried with installation but then i get:

Traceback (most recent call last):
  File "/usr/bin/mark2", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3018, in <module>
    working_set = WorkingSet._build_master()
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 614, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 627, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 805, in resolve
    raise DistributionNotFound(req)
pkg_resources.DistributionNotFound: urwid==1.1.1

And urwid is of course installed.

**Comment by barneygale** [comment]
try running ``export EDITOR=nano`` first.

**Comment by Yuri20** [comment]
And what it is supposed to do?

**Comment by gsand** [comment]
That sets your default editor to nano.

You can also create the directory /etc/mark2
and create a blank file inside called mark2.properties

**Comment by Yuri20** [comment]
And why I should have nano as defualt editor? I created `mark2.properties` in `/etc/mark2` and I while running `python2 mark2 config` I got:

Traceback (most recent call last):
  File "mark2", line 6, in <module>
    sys.exit(main())
  File "/home/jerzy/mc/mark2/mk2/launcher.py", line 611, in main
    command.start()
  File "/home/jerzy/mc/mark2/mk2/launcher.py", line 126, in start
    self.run()
  File "/home/jerzy/mc/mark2/mk2/launcher.py", line 485, in run
    subprocess.call([editor, path_tmp])
  File "/usr/lib/python2.7/subprocess.py", line 522, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1335, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory

**Comment by gsand** [comment]
No, you are doing it all wrong...

Check out the installation instructions again. follow them closely.

**Comment by Yuri20** [comment]
What I have done wrong? I clonned repository in `/home/jerzy/mc` and I have now it in `/home/jerzy/mc/mark2`. In installation guide is said that i don't have to install it in any particular place, so I left it in this directory i symlinked to `/usr/bin/mark2` by `sudo ln -s /home/jerzy/mc/mark2/mark2 /bin/mark2`. After symlinking INSTALL.md says nothing more, so i guess it's done. Which step of installation I described you I haven't done well?
